### DISCUSSION TITLE
lncc_loss gives too negative value back

### DISCUSSION THREAD

**Discussion by YuliangXiaoYLX** [Project-MONAI/MONAI#4467] [Q&A] [answered]
## Issue
I am working on Deep Atlas and using the local normalized cross correlation loss function (return -LNCC instead of 1-LNCC) to compute the similarity loss between warped moving image and fixed image. However, the loss is too big (too negative shown in the screenshot) and makes me confused.
## Result
![similarity_reg_losses](https://user-images.githubusercontent.com/70731556/172293409-9a2d90b6-a7d1-4e44-8e0c-d910c5269523.png)
The loss of each epoch is also saved as text file attached below.
[similarity_loss_reg.txt](https://github.com/Project-MONAI/tutorials/files/8850011/similarity_loss_reg.txt)

## Need Help
**Could anyone help me work on this confusion? I am appreciated your help, thanks!**

**Comment by ebrahimebrahim** [comment, marked as answer]
[A version of this discussion is here](https://github.com/Project-MONAI/MONAI/discussions/3463)
